packages/sources/css/src/components/tooltip/README.md

```diff
@@ -0,0 +1,134 @@
+# `@vtmn/css-tooltip`
+
+> Decathlon Design System - Vitamin specific CSS styles for text input component
+
+<a href="https://www.decathlon.design/726f8c765/p/595266-tooltip-beta/b/97ce1d">
+<img src="https://img.shields.io/badge/decathlon.design-docs-007dbc" alt="documentation" />
+</a>
+<a href="https://decathlon.github.io/vitamin-web/@vtmn/showcase-css/?path=/docs/components-tooltip--overview">
+<img src="https://img.shields.io/badge/storybook-css-d891bc?style=flat&logo=storybook" alt="storybook" />
+</a>
+<a href="https://www.npmjs.com/package/@vtmn/css-tooltip">
+<img src="https://img.shields.io/npm/v/@vtmn/css-tooltip?style=flat&logo=npm" alt="npm version" />
+</a>
+<a href="https://sonarcloud.io/dashboard?id=decathlon_vitamin-web_css">
+<img src="https://sonarcloud.io/api/project_badges/measure?project=decathlon_vitamin-web_css&metric=alert_status" alt="Quality Gate Status" />
+</a>
```

Comment:
You also need to push those information on the main `README.md` at the root of this repository.

Reply:
OK

Reply:
Done